niworkflows/interfaces/utils.py

```diff
@@ -202,8 +202,17 @@ def _gen_reference(fixed_image, moving_image, fov_mask=None, out_file=None,
         # out of the FoV.
         fixednii = nb.load(fixed_image)
         masknii = nb.load(fov_mask)
-        assert np.allclose(masknii.affine, fixednii.affine)
-        assert np.all(masknii.shape[:3] == fixednii.shape[:3])
+
+        if np.all(masknii.shape[:3] != fixednii.shape[:3]):
```

Comment:
Just a quick note: `img.shape[:3]` is a tuple, whose equality is all-or-nothing. So this is just `masknii.shape[:3] != fixednii.shape[:3]`.

Not worth cleaning up right now, but perhaps in passing at some point.